liquibase/changes/feat_622.sql

```diff
@@ -0,0 +1,28 @@
+DROP VIEW IF EXISTS location_with_translations;
+CREATE VIEW location_with_translations AS
```

Comment:
If we are creating a view for translations limited to _en_, let's reflect this on the name, something like `location_with_translations_en`.